modules/core/client/app/init.js

```diff
@@ -19,7 +19,7 @@ angular.module(ApplicationConfiguration.applicationModuleName).run(function ($ro
     if (toState.data && toState.data.roles && toState.data.roles.length > 0) {
       var allowed = false;
       toState.data.roles.forEach(function (role) {
```

Comment:
I'm leaving this for now. I don't think we need lodash on the client-side, just for this, either. 

I don't think there will be a huge performance increase, since we're talking about a small list of roles, that are being iterated. If we decide that we want refactor these loops, then I'd say we look at implementing something like ng-lodash; and do that as a sweeping change.